### ISSUE TITLE
Tab items are not accessible with keyboard and content is also not visible properly.

### ISSUE THREAD

**Issue by msftedad** [microsoft/fluentui-blazor#3825]
# 🐛 Bug Report

Tab items are not accessible with keyboard and content is also not visible properly.
Users who rely on keyboard navigation will get impacted as they cannot access the tab items or view the associated content properly, leading to significant accessibility and usability barriers.

## 💻 Repro or Code Sample

**Steps to Repro:**

1. Open URL: [Tabs - Fluent UI Blazor library](https://www.fluentui-blazor.net/Tabs) in latest edge browser.
2. Press tab key till Custom Header and Panel usage "Tab Items".
3. Inspect and adjust the width to "150 px".
4. Verify whether Tab items are accessible with keyboard and content is also visible properly or not.

## 🤔 Expected Behavior
Tab items should be accessible with keyboard and content is also should be visible properly.

## 😯 Current Behavior
Tab items are not accessible with keyboard and content is also not visible properly.

<img width="960" alt="Image" src="https://github.com/user-attachments/assets/a75e477c-20f5-4659-9136-0729b2f22a7b" />

## 🔦 Context

Users who rely on keyboard navigation will get impacted as they cannot access the tab items or view the associated content properly, leading to significant accessibility and usability barriers.
Tab items should be accessible with keyboard and content is also should be visible properly.

## 🌍 Your Environment

* OS & Device: Windows 11 Enterprise Insider Preview Version 24H2(OS Build 26100.4061)
* Browser Edge Version: 136.0.3240.76 (Official build) (64-bit)
* .NET Version [Powered by .NET 10.0.0-preview.3.25171.5](https://dotnet.microsoft.com/en-us/learn/aspnet/what-is-aspnet-core) and Fluent UI Blazor library Version 4.11.9+73d35871

**Comment by msftedad** [comment]
Hi @vnbaaij, In this issue, when at least one item is visible, we’re able to navigate to others using the left/right arrow keys—but the value is still not displayed. When no options are visible, navigation isn’t possible at all. Could you please reopen this issue as it needs to be fixed?

**Comment by vnbaaij** [comment]
We are not going to (be able to) fix this. The overflow is something we bolted on top of the v2 Tabs web component and that can't handle interaction between the tabs if the # width is too small.

PS please refrain from saying things 'needs to be fixed'. That is not how an open source project works.
